clients/fossid-webapp/src/main/kotlin/model/result/MatchType.kt

```diff
@@ -20,6 +20,7 @@
 package org.ossreviewtoolkit.clients.fossid.model.result
 
 enum class MatchType {
+    ERROR,
```

Comment:
BTW, I believe you've clarified with FOSSID by now what these enums exactly mean. So would you mind adding according docs for all of these (also for `ScanState`) in another commit?

Reply:
We clarified only the `ScanState` therefore I'll add documentation only for it.